src/libs/ErrorUtils.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Returns a user friendly error message for the respective error code from the API
+ *
+ * e.g.
+ *
+ * getErrorMessageFromErrorCode(401)  =>  Incorrect login or password. Please try again.
+ *
+ * @param {Number} errorCode
+ *
+ * @returns {String}
+ */
+function getErrorMessageFromErrorCode(errorCode) {
```

Comment:
I would just place this in the context of `Authenticate`. Placing this here implies these error messages and codes are generic. They're not. They're only relevant in the context of `Authenticate`